bin/rose-test-battery

```diff
@@ -40,6 +40,7 @@
 . $(dirname $0)/../lib/bash/rose_init
 . $(dirname $0)/../lib/bash/rose_log
 rose_init
+mkdir -p ~/.metomi
```

Comment:
I guess there's a problem somewhere in our code if you don't have a .metomi directory then?